managed/devops/bin/yb_platform_util_py2.py

```diff
@@ -0,0 +1,498 @@
+    """
+    Get details of the ongoing task.
+
+    :param task_id: Task UUID.
+    :param base_url: base url of the platform back end.
+    :param customer_uuid: authentication token for the customer.
+    :param auth_uuid: authentication token for the customer.
+    :return: None
+    """
+    task_url = base_url + "/api/v1/customers/" + customer_uuid + "/tasks/" + task_id
+    response = call_api(task_url, auth_uuid)
+    universe_json = convert_unicode_json(json.loads(response.read()))
+    if universe_json["status"] == "Running":
+        response = {"data": int(universe_json["percent"]), "status": "success", "error": ""}
+        print(response)
+    elif universe_json["status"] == "Success":
+        response = {"data": 100, "status": "success", "error": ""}
+        print(response)
+    elif universe_json["status"] == "Failure":
+        content = {
+            "message": "{0} failed".format(universe_json["title"]),
+            "details": universe_json["details"]
+        }
+        response = {"data": content, "status": "success",
+                    "error": content}
+        print(response)
+
+
+def create_universe_config(universe_data, universe_name):
+    """
+    Create the universe config data from the json file.
+
+    :param universe_data: Stored universe data.
+    :param universe_name: Name of the new universe to be created.
+    :return: Configured universe json.
+    """
+    configure_json = {}
+    clusters = universe_data['universeDetails']['clusters']
+    user_az_selected = universe_data['universeDetails']['userAZSelected']
+
+    excluded_keys = [
+        "uuid",
+        "storageType",
+        "awsArnString",
+        "useHostname",
+        "preferredRegion",
+        "regions",
+        "index",
+        "placementInfo"
+    ]
+
+    clusters_list = get_cluster_list(clusters, excluded_keys, universe_name)
+    configure_json["clusters"] = clusters_list
+    configure_json["clusterOperation"] = "CREATE"
+    configure_json["userAZSelected"] = user_az_selected
+    configure_json["currentClusterType"] = "PRIMARY"
+
+    return configure_json
+
+
+def get_cluster_list(clusters, excluded_keys, universe_name):
+    """
+    Helper method for creating universe config which returns modified cluster list.
+
+    :param clusters: List of clusters.
+    :param excluded_keys: Keys to be excluded
+    :param universe_name: Name of the new universe to be created.
+    :return: cluster list.
+    """
+    clusters_list = []
+    for each_cluster in clusters:
+        cluster = {}
+        for key, val in each_cluster.items():
+            if key not in excluded_keys:
+                user_intent = {}
+                if key == "userIntent":
+                    if universe_name:
+                        val["universeName"] = universe_name
+                    for key1, val1 in val.items():
+                        if key1 not in excluded_keys:
+                            if key1 == "deviceInfo":
```

Comment:
Looks slightly strange for me. Please add comments what and why are your doing here.